discord/app_commands/commands.py

```diff
@@ -2094,6 +2094,22 @@ def describe(**parameters: Union[str, locale_str]) -> Callable[[T], T]:
         async def ban(interaction: discord.Interaction, member: discord.Member):
             await interaction.response.send_message(f'Banned {member}')
 
+    Alternatively, you can describe parameters using Google, Sphinx, or Numpy style docstrings.
+
+    Example:
+
+    .. code-block:: python3
+
+        @app_commands.command()
+        async def ban(interaction: discord.Interaction, member: discord.Member):
+            '''
```

Comment:
The first line of docstring is used as the command's description. You should also add it.
```suggestion
            """Bans a member

```